docs/fundamentals/integrations-overview.md

```diff
@@ -14,35 +15,53 @@ ms.topic: conceptual
+Client integrations define configuration schema, wire up client libraries to dependency injection (DI), and add health checks, resiliency, and telemetry where applicable. These packages configure existing client libraries to connect to hosting integrations. They extend the <xref:Microsoft.Extensions.Hosting.IHostApplicationBuilder> interface allowing client-consuming projects, such as your web app or API, to use the connected resource. The official [client integration NuGet packages](https://www.nuget.org/packages?q=owner%3A+aspire+tags%3A+aspire+client+integration&includeComputedFrameworks=true&prerel=true&sortby=relevance) are tagged with `aspire`, `integration`, and `client`.
 
 For more information on creating a custom client integration, see [Create custom .NET Aspire client integrations](../extensibility/custom-client-integration.md).
 
 ### Relationship between hosting and client integrations
 
-Hosting and client integrations are best when used together, but are **not** coupled and may be used separately. Some hosting integrations may not have a corresponding client integration. Configuration is what makes the hosting integration work with the client integration.
+Hosting and client integrations are best when used together, but are **not** coupled and can be used separately. Some hosting integrations don't have a corresponding client integration. Configuration is what makes the hosting integration work with the client integration.
 
 Consider the following diagram that depicts the relationship between hosting and client integrations:
 
 :::image type="content" source="media/integrations-thumb.png" lightbox="media/integrations.png" alt-text="A diagram ":::
 
 The app host project is where hosting integrations are used. Configuration, specifically environment variables, is injected into projects, executables, and containers, allowing client integrations to connect to the hosting integrations.
 
-## Available integrations
+## Client integration features
 
-The following section details the available .NET Aspire integrations, links to their respective NuGet packages, and provides a brief description of each integration.
+When you add a client integration to a project within your .NET Aspire solution, [service defaults](service-defaults.md) are automatically applied to that project; meaning the Service Defaults project is referenced and the `AddServiceDefaults` extension method is called. These defaults are designed to work well in most scenarios and can be customized as needed. The following service defaults are applied:
+
+- **[Service discovery](../service-discovery/overview.md)**: Automatically locates services within a distributed application. Service discovery is a key integration of microservice architectures.
```

Comment:
This isn't really client integration feature. We should replace this with how client integrations have a configuration schema and integration with dependency injection.
```suggestion
```

Reply:
Perhaps this section should be renamed to "Integration features" as we're really talking about both ends, no? Service discovery is required on both ends, right?

Reply:
ish, it's a little fuzzy because integrations don't really use service discovery. I think we should remove this and talk about configuration instead. That's a key part of these integrations.